src/rtmp/rtmppacketizer.cpp

```diff
@@ -205,11 +206,15 @@ std::unique_ptr<VideoFrame> RTMPH26xPacketizer<DescClass, SPSClass, PPSClass, co
 		//Set timestamp
 		set8(sei, 19, videoFrame->GetSenderTime());
 
+		//Escape nal
+		uint8_t seiEscaped[sizeof(sei)*2];
+		auto seiSize = NalEscapeRbsp(seiEscaped, sizeof(seiEscaped), sei, sizeof(sei)).value();
+
 		//Append nal
-		auto ini = frame->AppendMedia(sei, sizeof(sei));
+		auto ini = frame->AppendMedia(seiEscaped, seiSize);
```

Comment:
seiSize is an optional, wouldn't this be? 
```suggestion
		auto ini = frame->AppendMedia(seiEscaped, *seiSize);
```

Reply:
also, we should check returned value

Reply:
ok, missed the `value()`